parcels/particleset.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, fieldset, pclass=JITParticle, lon=None, lat=None, depth=None,
         self.fieldset = fieldset
         self.fieldset.check_complete()
         partitions = kwargs.pop('partitions', None)
+        self.aborted = False  # TODO is this necessary?
```

Comment:
Check this `#TODO` message? Is it indeed necessary?

Reply:
the variable IS needed, the TODO message not. I may need to go over Reint's smaller changes still ...